libraries/scripting/common/src/kotlin/script/experimental/api/scriptCompilation.kt

```diff
@@ -148,7 +148,7 @@ val ScriptCompilationConfigurationKeys.refineConfigurationOnAnnotations by Prope
 val ScriptCompilationConfigurationKeys.refineConfigurationBeforeCompiling by PropertiesCollection.key<List<RefineConfigurationUnconditionallyData>>(isTransient = true)
 
 /**
- * The list of script fragments that should be compiled intead of the whole text
+ * The list of script fragments that should be compiled instead of the whole text
```

Comment:
Please, to the separate commit (with the "[minor]" prefix), it is completely irrelevant here.

Reply:
Done